### PR TITLE
Be less agressive and more patient when creating large master.

### PR DESCRIPTION
**Pull request by porridge**
**What this PR does / why we need it**:

Workaround for #55777

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by bowei** [comment]
Can you explain how this will address the linked issue? If the zone is out of quota, wouldn't waiting long not change anything?

**Comment by porridge** [comment]
@bowei see https://github.com/kubernetes/kubernetes/issues/55777#issuecomment-344545873 - waiting actually did make a difference, because it's not so much about quota, as about binpacking the large master. In the background we're also working towards a better solution, but I think this workaround might be good enough for the time being.

File: cluster/gce/gci/master-helper.sh [hunk 1]
```diff
@@ -76,6 +76,12 @@ function replicate-master-instance() {
 function create-master-instance-internal() {
   local gcloud="gcloud"
   local retries=5
+  local sleep_sec=10
+  if [[ "${MASTER_SIZE}" =~ -64$ ]]; then
```
Comment (bowei) [inline cluster/gce/gci/master-helper.sh]: Is this the biggest MASTER_SIZE?
maybe?
```
local suffix=$( echo "${MASTER_SIZE}" | sed 's/.*-\([0-9]\+\)/\1' )
if [[ "${suffix}" -ge 64 ]]; then
```

Comment (porridge) [reply cluster/gce/gci/master-helper.sh]: Good point, this will be more robust.
Done.

**Comment by porridge** [comment]
Rebased.

**Comment by porridge** [comment]
/retest
crossing fingers

**Comment by shyamjvs** [comment]
/kind bug
/priority important-soon
/sig scalability
(as this should reduce flakiness in release-blocking scalability jobs due to #55777 )

**Pull request merged**